index.html

```diff
@@ -6102,21 +6102,30 @@ <h2 id="AudioWorkletNodeOptions">
             <dl title="dictionary AudioWorkletNodeOptions : AudioNodeOptions"
             class="idl">
               <dt>
-                unsigned long numberOfInputs
+                unsigned long numberOfInputs = 1
               </dt>
               <dd>
                 This is used to initialize the value of <a>AudioNode</a>
                 <a href="#widl-AudioNode-numberOfInputs">numberOfInputs</a>
                 attribute.
               </dd>
               <dt>
-                unsigned long numberOfOutputs
+                unsigned long numberOfOutputs = 1
               </dt>
               <dd>
                 This is used to initialize the value of <a>AudioNode</a>
                 <a href="#widl-AudioNode-numberOfOutputs">numberOfOutputs</a>
                 attribute.
               </dd>
+              <dt>
+                record&lt;DOMString, double&gt; parameterData
+              </dt>
+              <dd>
+                This is a list of user-defined key-value pairs, that is used to
```

Comment:
s/pairs, that is/pairs that are/

(Not 100% sure about "are". Can't figure out if "that" refers to list or pairs)